src/scss/02-layout/_menu-layout-native.scss

```diff
@@ -29,6 +29,24 @@
 // Responsice --------------------------------------------------------------------
 ///
 .desktop {
+	.layout {
+		&.layout-native {
+			&.layout-side {
+				&.aside-expandable {
+					.app-menu-content {
+						padding-top: calc(var(--header-size) + var(--header-size-content) + var(--os-safe-area-top));
+					}
```

Comment:
shouldn't the .hide-header-on-scroll:not(.header-is--visible) be here?

Reply:
When the layout were created and passed the css to SCSS all of this was tested. At this task, I doesn't change anything on that. I just moved and reorganized the existing rules.

Reply:
the code is repeated.... is just what I'm saying

Reply:
Reviewed and fixed. Now are concated and with the right rule